gulpfile.js

```diff
@@ -304,31 +304,17 @@ gulp.task('server:backend', function() {
 gulp.task('server:frontend', ['generate_index'], function(done) {
     var connect = require('gulp-connect');
     app = connect.server({
+        port: 8700,
         root: 'compair/static',
         livereload: false, // set to false, otherwise gulp will not exit
         middleware: function(connect, o) {
             var url = require('url');
             var proxy = require('proxy-middleware');
             return [ (function() {  // rewrite all requests against /app to /
                 // because generated index.html has a base /app
-                var options = url.parse('http://localhost:8080/');
+                var options = url.parse('http://localhost:8700/');
                 options.route = '/app';
                 return proxy(options);
-            })(),
-            (function() {  // proxy api calls
```

Comment:
we don't need those proxies?

Reply:
no, the http_backend_mocks catch all api calls so the proxies are never actually used (actually the port 8081 was never being called by the tests anyways so it was doubly not being used)